### DISCUSSION TITLE
Monitor Stuck on "Allow zero brightness via software or combined dimming"

### DISCUSSION THREAD

**Discussion by mfschafer3** [MonitorControl/MonitorControl#1405] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

I checked this setting after initially downloading the app, but despite unchecking it it seems to be stuck on this setting. 

### Expected behavior

I will be able to change the brightness setting instead of it being stuck in zero brightness. 

### Environment Information (please complete the following information)

```markdown
- macOS version:
- Mac model:
- MonitorControl version:
- Monitor(s):
- Monitor Cable(s)/Connection(s):
- Apple Silicon/M1 (yes or no):
```

**Comment by waydabber** [comment]
The feature is about allowing zero brightness (aka. full blank screen) but enabling it should not make the display to actually stuck on zero brightness (full dimming). If your screen is stuck at a blank screen then something else is not right.
